duneprototypes/Protodune/singlephase/DetectorServices/Providers/DetectorPropertiesProtoDUNEsp.cxx

```diff
@@ -340,7 +335,7 @@ namespace spdp{
     // Calculate kinematic quantities.
 
     double bg = mom / mass;           // beta*gamma.
-    double gamma = sqrt(1. + bg*bg);  // gamma.
+    double gamma = square(1. + bg*bg);  // gamma.
```

Comment:
Are you sure we're not introducing a math mistake here?

Reply:
Good catch, @aolivier23!  Yes, this change is a mistake.  Are you able to revert it?